src/terminal/input/terminalInput.cpp

```diff
@@ -19,7 +19,7 @@
 
 using namespace Microsoft::Console::VirtualTerminal;
 
-DWORD const dwAltGrFlags = LEFT_CTRL_PRESSED | RIGHT_ALT_PRESSED;
+constexpr DWORD dwAltGrFlags = LEFT_CTRL_PRESSED | RIGHT_ALT_PRESSED;

```

Comment:
This `dwAltGrFlags` is not actually used anywhere else! Wonder what happened to it during the past year.

Reply:
If it isn't used anymore, then maybe we should just remove it entirely